WeakAuras.lua

```diff
@@ -1810,6 +1810,28 @@ function WeakAuras.Add(data)
   -- end
 end
 
+
+function removeSpellNames(data)
```

Comment:
Yeah I know, we should decide on one standard one day, I think only transmission.lua is 4 spaces, what do you prefer?